app/views/teacher_training_adviser/steps/_overseas_callback.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <div class="govuk-form-group">
   <%= f.label :phone_call_scheduled_at, "Select your preferred day and time for a callback", class: "govuk-label" %>
-  <%= f.select :phone_call_scheduled_at, format_booking_quotas(TeacherTrainingAdviser::Steps::OverseasCallback::grouped_quotas, @wizard.export_data["time_zone"]), {}, { class: "govuk-select" } %>
+  <%= f.select :phone_call_scheduled_at, format_booking_quotas(GetIntoTeachingApiClient::CallbackBookingQuotasApi.new.get_callback_booking_quotas, @wizard.export_data["time_zone"]), {}, { class: "govuk-select" } %>
```

Comment:
Whats the logic for moving data layer work into the View? I'd prefer to see this remain in the data model. This means it can be tested more easily, and if additional manipulation on the data is required in a future iteration, it can then be done in the model?

Reply:
Yeah it should be - it's a bit mixed in the app as a whole right now; I created a ticket last week to go through it all and make it consistent https://dfedigital.atlassian.net/browse/GITPB-646